### ISSUE TITLE
ip_address get null with Laravel 5.5

### ISSUE THREAD

**Issue by dungnh** [cyrildewit/laravel-page-view-counter#39]
- Laravel Page View Counter Version: 1.0.3
- Laravel Version: 5.5.#
- PHP Version: 7.1
- Database Driver & Version: MySQL 5.7

### Description:
Thank you for a great package. I installed it and everything seems works well, but I got ip_address is null in DB. Tried to check in HasPageViewCounter Trait and it's using `$newView->ip_address = (new Request)->ip();`, I think it has problem with Laravel 5.5, could you check for help, please? 

Thank you.

**Comment by cyrildewit** [comment]
Hi, sorry for the late response.

Could you replace `(new Request)->ip();` in the HasPageViewCounter trait to `Request::ip();`. I will create a new release if this solves the problem.

**Comment by Mobius1** [comment]
This worked for me:

Old
```php
$newView->ip_address = (new Request)->ip();
```

New:
```php
$newView->ip_address = \Request::ip();
```

**Comment by cyrildewit** [comment]
Thanks for checking @Mobius1!

I just comitted the fix to the v1.0.x branch. I will release v1.0.4 after the Travis CI builds passed.
